openstack/cinder/templates/backup-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+      annotations:
+        configmap-etc-hash: {{ include (print $.Template.BasePath "/etc-configmap.yaml") . | sha256sum }}
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (lt .Capabilities.KubeVersion.Minor "5") }}
+        scheduler.alpha.kubernetes.io/affinity: >
+            {{ tuple . "cinder" "backup" | include "kubernetes_pod_anti_affinity" }}
+{{- end }}
+    spec:
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (ge .Capabilities.KubeVersion.Minor "7") }}
+{{ tuple . "cinder" "backup" | include "kubernetes_pod_anti_affinity" | indent 6 }}
+{{- end }}
+      containers:
+        - name: cinder-backup
+          image: {{.Values.global.imageRegistry}}/{{.Values.global.image_namespace}}/ubuntu-source-cinder-backup:{{.Values.imageVersionCinderVolumeBackup | default .Values.imageVersion | required "Please set cinder.imageVersion or similar" }}
+          imagePullPolicy: IfNotPresent
+          command:
+            - kubernetes-entrypoint
+          env:
+            - name: COMMAND
+              value: "cinder-backup"
+            - name: NAMESPACE
+              value: {{ .Release.Namespace }}
+            - name: DEPENDENCY_SERVICE
+              value: "cinder-postgresql,rabbitmq"
+            - name: SENTRY_DSN
+              value: {{.Values.sentry_dsn | quote}}
+            - name: PGAPPNAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          livenessProbe:
+            httpGet:
+              path: /
+              port: cinder-backup
+            initialDelaySeconds: 15
+            timeoutSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /healthcheck
+              port: cinder-backup
+            initialDelaySeconds: 15
+            timeoutSeconds: 5
+          volumeMounts:
+            - name: cinder-etc
+              mountPath: /tmp/cinder-backup.sh
+              subPath: cinder-backup.sh
+              readOnly: true
+            - name: etccinder
+              mountPath: /etc/cinder
+            - name: cinder-etc
+              mountPath: /etc/cinder/cinder.conf
+              subPath: cinder.conf
+              readOnly: true
+            - name: cinder-etc
+              mountPath: /etc/cinder/api-paste.ini
+              subPath: api-paste.ini
+              readOnly: true
+            - name: cinder-etc
+              mountPath: /etc/cinder/policy.json
+              subPath: policy.json
+              readOnly: true
+            - name: cinder-etc
+              mountPath: /etc/cinder/logging.ini
+              subPath: logging.ini
+              readOnly: true
+            - name: cinder-etc
+              mountPath: /etc/cinder/cinder_audit_map.yaml
+              subPath: cinder_audit_map.yaml
+              readOnly: true
+      volumes:
+        - name: etccinder
+          emptyDir: {}
+        - name: cinder-etc
+          configMap:
+            name: cinder-etc
```

Comment:
We need a configmap for the cinder-backup-<availability-zone>, containing the file "cinder-backup.conf", holding two config values

```
[DEFAULT]
host = cinder-backup-{{ az}}
storage_availability_zone = {{ region }}{{ az}}
```

Reply:
That is, additionally to the cinder-etc configmap